eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/ConsoleCacheWarmupPass.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the ConsoleCacheWarmupPass class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Cache related compiler pass.
+ *
+ * Ensures class cache warmup is disabled in console mode.
+ */
+class ConsoleCacheWarmupPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        if (PHP_SAPI !== 'cli' ||
```

Comment:
Maybe add a comment here on why we only opt into this pass on CLI.

Reply:
Right, added.